### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by PerLysemose**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Feature_Request.md [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
+The proposed measurable benefit to the end user or business: "We believe this [business outcome] will be achieved if [these users/actors/society] successfully achieve [this user/actors/society outcome] with [this feature]"
+
+## Acceptance criteria and must have scope
+Define what is a must-have for launch and in-scope. Keep this section fluid and dynamic until you lock-in priority during planning.
+
+## Objectives and key results
+What is the revenue impact on our business? Is there a potential penalty or other adverse consequences if we delay?
+
+## Leading Indicators
+What is the revenue impact on our business? Is there a potential penalty or other adverse consequences if we delay?
```
Comment (prtandrup) [inline .github/ISSUE_TEMPLATE/Feature_Request.md:23]: This is the same text as Objectives and key results, perhaps a copy/paste error? I've added the original template text from AzDo.

```suggestion
An indicator of performance that might predict future success
```

File: .github/ISSUE_TEMPLATE/Feature_Request.md [hunk 2]
```diff
@@ -1,18 +1,31 @@
...
+title: ''
+labels: epic, feature
+assignees: ''
+
 ---
 
-### Is your feature request related to a problem? Please describe.
-<!--- A clear and concise description of what the problem is. Ex. I have an issue when [...] --->
+## Problem Description
+New Product: “The current state of the [domain] has focused primarily on [customer segments, pain points, etc]. What existing products/services fail to address is [this gap] Our product/service will address this gap by [vision/strategy] Our initial focus
```
Comment (prtandrup) [inline .github/ISSUE_TEMPLATE/Feature_Request.md:11]: This section seems incomplete, how about ending it like this?

```suggestion
New Product: “The current state of the [domain] has focused primarily on [customer segments, pain points, etc]. What existing products/services fail to address is [this gap] Our product/service will address this gap by [vision/strategy] Our initial focus ..."
```

File: .github/ISSUE_TEMPLATE/Feature_Request.md [hunk 3]
```diff
@@ -1,18 +1,31 @@
...
+What is the revenue impact on our business? Is there a potential penalty or other adverse consequences if we delay?
 
-### Describe the solution you'd like.
-<!--- A clear and concise description of what you want to happen. Add any considered drawbacks. --->
+## Non Functional Requirements
 
-### Describe alternatives you've considered.
-<!--- A clear and concise description of any alternative solutions or features you've considered. --->
+## Stakeholders
+Describe who needs to be kept up-to-date about this feauture, included in discussions, or updated along the way. Stakeholders can be both in Product/Engineering, as well as other teams like Customer Success who might want to keep customers updated on the feature.
```
Comment (prtandrup) [inline .github/ISSUE_TEMPLATE/Feature_Request.md:28]: To me, the last part sounds more as a general template and not so related to GEH/DH. Perhaps just keep the text like this:
```suggestion
Describe who needs to be kept up-to-date about this feauture, included in discussions, or updated along the way. 
```

**Review by prtandrup — COMMENTED** [review]
A few comments :)

**Comment by BjarkeMeier** [comment]
@Renetnielsen, would you like us to merge this PR?

**Comment by PerLysemose** [comment]
Not relevant anymore.

**Pull request closed without merging**